rocAL/rocAL/source/master_graph.cpp

```diff
@@ -224,7 +224,7 @@ void
 MasterGraph::decrease_image_count()
 {
     if(!_loop)
-        _remaining_images_count -= _user_batch_size;
+        _remaining_images_count -= ((_sequence_rearrange_batch_decrementer > 0)? _sequence_rearrange_batch_decrementer:_user_batch_size);
```

Comment:
need better name for _sequence_rearrange_batch_decrementer

Reply:
Okay.